README.md

```diff
@@ -1,104 +1,159 @@
-# nucypher-ts
+Full Documentation can be found [here](https://docs.threshold.network/app-development/threshold-access-control-tac).
+# Get Started with TAC
 
-Communicate with NuCypher network from browser.
+The following interactive tutorial is a quick way for developers to familiarize themselves with the Threshold Access Control service. Before jumping into the following, we recommend reading this introduction to [Conditions Based decryption](https://docs.threshold.network/fundamentals/threshold-access-control/conditions-based-decryption-cbd) and its key concepts. CBD is the primary technology underpinning Threshold Access Control and directly enables data sharing predicated on the fulfillment of predefined conditions.
 
-## Disclaimer
+## 1. Install nucypher-ts
 
-**This is a work in progress**
+We'll kick things off by installing `nucypher-ts` – a TypeScript library for interacting with access control functionality in the browser. The APIs for leveraging most TAC functionality are contained in nucypher-ts.
 
-- SDK does not support policy revocation.
-- We expect breaking changes.
```

Comment:
Do we want to get rid of this disclaimer?

Reply:
I see there is a new disclaimer [a few lines below](https://github.com/nucypher/nucypher-ts/pull/117/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R15).